spec/index.bs

```diff
@@ -7444,6 +7452,41 @@ GPUSwapChain includes GPUObjectBase;
         1. Return |texture|.
 </div>
 
+
+## <dfn dfn-type=enum-value dfn-for=GPUSwapChainAlphaMode>GPUSwapChainAlphaMode</dfn> ## {#GPUSwapChainAlphaMode}
+
+This enum selects how the swap chain canvas will paint onto the page.
+
+<table class='data'>
+    <thead>
+        <tr>
+            <th>GPUSwapChainAlphaMode
+            <th>Description
+            <th>dst.rgb
+            <th>dst.a
+    </thead>
+    <tr>
+        <td>{{GPUSwapChainAlphaMode/opaque}}
+        <td>Paint RGB as opaque and ignore alpha values.
```

Comment:
We need to figure out how to describe or implement it consistently. If on one platform we have to clear alpha to 1 at the end of the pass, and on others we don't then an application that reads back the swapchain will see the difference.

Reply:
We (kvark/jdashg/I) discussed this and agreed that it wouldn't be possible to do this on a pass. It has to occur after user code loses access to the texture (after the end of the frame), so it would have to be a separate pass.